sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/analysis/TestRelations.scala

```diff
@@ -62,6 +62,9 @@ object TestRelations {
   val listRelation = LocalRelation(
     AttributeReference("list", ArrayType(IntegerType))())
 
+  val nestedListRelation = LocalRelation(
```

Comment:
will this relation be used in more tests? if not let's inline it in your test case.

Note that, the more common test relations we have, the harder to review test cases, as we need to search the definition of the used test relations.

Reply:
ok, I'll remove this.